src/styles/blocks/page.scss

```diff
@@ -0,0 +1,5 @@
+.page {
```

Comment:
Are you sure that you are using this styling somewhere? If not it can be removed